python/mxnet/optimizer.py

```diff
@@ -157,7 +157,7 @@ def set_wd_mult(self, args_wd_mult):
         """
         self.wd_mult = {}
         for n in self.idx2name.values():
-            if not n.endswith('_weight'):
+            if not (n.endswith('_weight') or n.endswith('_gamma')):
```

Comment:
@vchuravy yes, this allows to set learning rate easily for each parameter. Do you have time to sync this feature to Julia? I might have some time to do some porting to julia after the end of this semester, but currently quite busy.